generator-fraudulentlogic/generators/index.js

```diff
@@ -0,0 +1,21 @@
+var Generator = require('yeoman-generator');
+
+module.exports = class extends Generator {
+  async prompting() {
+    this.answers = await this.prompt([
```

Comment:
whats with the title "fraudulentlogic"?

Reply:
@reubenae can you suggest some better name for this generator, I couldn't come up with a good name

Reply:
@reubenae @jitinmaher How does generator-fraudpreventiondata sound?